rules/experimental/rbe/exec_properties.bzl

```diff
@@ -156,3 +246,107 @@ def merge_dicts(*dict_args):
     for dictionary in dict_args:
         result.update(dictionary)
     return result
+
+def _exec_property_sets_repository_impl(repository_ctx):
+    repository_ctx.file(
+        "BUILD",
+        content = "",
+        executable = False,
+    )
+    repository_ctx.file(
+        "constants.bzl",
+        content = repository_ctx.attr.constants_bzl_content,
+        executable = False,
+    )
+
+# _exec_property_sets_repository is a repository rule that creates a repo with the specified exec_property_sets.
+_exec_property_sets_repository = repository_rule(
+    implementation = _exec_property_sets_repository_impl,
+    local = True,
+    attrs = {
+        "constants_bzl_content": attr.string(
+            mandatory = True,
+            doc = "The content of the constants.bzl file within the repository rule.",
+        ),
+    },
+)
+
+def _verify_dict_of_dicts(name, dicts):
+    """ Verify that dict is of type {string->{string->string}}.
+
+    Args:
+      name: Name of the repo rule. Used for error messages.
+      dicts: a dict whose key is a string and whose value is a dict from string to string.
+    """
+
+    # Verify that dict is of type {string->{string->string}}.
+    for key, value in dicts.items():
+        if type(key) != "string":
+            fail("In repo rule %s, execution property set name %s must be a string" % (name, key))
+        if type(value) != "dict":
+            fail("In repo rule %s, execution property set of %s must be a dict" % (name, key))
+        for k, v in value.items():
+            if type(k) != "string":
+                fail("In repo rule %s, execution property set %s, the key %s must be a string" % (name, key, k))
+            if type(v) != "string":
+                fail("In repo rule %s, execution property set %s, key %s, the value %s must be a string" % (name, key, k, v))
+
+def custom_exec_properties(name, dicts):
+    """ Creates a repository containing execution property dicts.
+
+    Use this macro in your WORKSPACE.
+
+    Args:
+      name: Name of the repo rule.
+      dicts: The execution property set constants.
+    """
+    _verify_dict_of_dicts(name, dicts)
+
+    constants_bzl_content = ""
+    for key, value in dicts.items():
+        constants_bzl_content += "%s = %s\n" % (key, value)
+
+    _exec_property_sets_repository(
+        name = name,
+        constants_bzl_content = constants_bzl_content,
+    )
+
+STANDARD_PROPERTY_SETS = {
```

Comment:
Add a short doc comment mentioning this is the SoT for the properties used by RBE

Reply:
Is there a pointer to somewhere that explains each of these? add pointer here if it exists, document properties here if it does not.

Reply:
Done.
Added a comment saying that this is the SoT for the list of standard constants.
As for the docs of the specific properties, I added a link to https://cloud.google.com/remote-build-execution/docs/remote-execution-environment#remote_execution_properties.
This link currently contains only the docker related properties, but we should update our RBE docs to contain all the properties.
We made a decision *not* to duplicate the documentation of each property for the sake of easier maintenance. We also decided to have the property names as received by create_exec_properties_dict() match exactly the RBE names with the only difference being that they are in snake_case.